Pod/Classes/WPBadgeView.m

```diff
@@ -79,15 +76,13 @@ - (void)setInsets:(UIEdgeInsets)insets
     self.bottomConstraint.constant = -insets.bottom;
     self.leadingConstraint.constant = insets.left;
     self.trailingConstraint.constant = -insets.right;
-    [self setNeedsLayout];
```

Comment:
Are you sure this isn't necessary any more? Or perhaps `[self layoutIfNeeded]`?

Reply:
I tested it with:
```
    cell.badgeView.cornerRadius = 24;
    cell.badgeView.insets = UIEdgeInsetsMake(15, 15, 15, 15);
```
From `cellForItemAtIndexPath` and it worked. So it seems that it's not necessary.

![img_0003](https://user-images.githubusercontent.com/9772967/40801659-77af9d2e-64e1-11e8-84b6-5686f34edf70.PNG)

But you are right, in other use cases it might be necessary so I will add the `[self layoutIfNeeded]` call.
 
Thanks for pointing it out!